newthread.php

```diff
@@ -160,7 +160,7 @@
 $maximageserror = $attacherror = '';
 
 // Handle attachments if we've got any.
-if($mybb->settings['enableattachments'] == 1 && ($mybb->get_input('newattachment') || $mybb->get_input('updateattachment') || ((($mybb->input['action'] == "do_newthread" && $mybb->get_input('submit')) || ($mybb->input['action'] == "newthread" && isset($mybb->input['previewpost'])) || isset($mybb->input['savedraft'])) && $_FILES['attachments'])))
+if($mybb->settings['enableattachments'] == 1 && ($mybb->get_input('newattachment') || $mybb->get_input('updateattachment') || ((($mybb->input['action'] == "do_newthread" && $mybb->get_input('submit')) || ($mybb->input['action'] == "newthread" && isset($mybb->input['previewpost'])) || isset($mybb->input['savedraft'])) && (isset($_FILES['attachments'])?$_FILES['attachments']:false))))
```

Comment:
As per [MyBB Development Standards](https://docs.mybb.com/1.8/development/standards/):
> Ternary operators are not permitted and should not be used.
> Control structures should be kept as simple as possible and large nested if structures should also be avoided.
> Banned Practices: Use of ternary operators

The existing code is already bad and difficult to understand but adding a ternary operator makes it way worse. Wouldn't a simple `empty` do exactly the same?